packages/components/src/components/vertical-tab/vertical-tab-header.tsx

```diff
@@ -29,9 +64,9 @@ const VerticalTabHeader = ({
     onChange,
     selected,
     selectedKey = 'label',
-}) => {
+}: React.PropsWithChildren<TVerticalTabHeader>) => {
     const label = item.label || item.title || item.getTitle?.();
```

Comment:
```suggestion
    const label = item.label || item.title || item.getTitle?.() || '';
```
@yauheni-deriv in case you have a warning at line 71 like i do which says that _Argument **label** of type 'string | undefined' is not assignable to parameter of type 'string'_.